pvsite_datamodel/sqlmodels.py

```diff
@@ -155,12 +154,12 @@ class SiteSQL(Base, CreatedMixin):
         comment="Auto-incrementing integer ID of the site for use in ML training",
     )
 
-    forecasts: List["ForecastSQL"] = relationship("ForecastSQL", back_populates="site")
-    generation: List["GenerationSQL"] = relationship("GenerationSQL")
-    inverters: List["InverterSQL"] = relationship(
+    forecasts: Mapped["ForecastSQL"] = relationship("ForecastSQL", back_populates="site")
+    generation: Mapped["GenerationSQL"] = relationship("GenerationSQL")
+    inverters: Mapped["InverterSQL"] = relationship(
         "InverterSQL", back_populates="site", cascade="all, delete-orphan"
     )
-    site_groups: List["SiteGroupSQL"] = relationship(
+    site_groups: Mapped["SiteGroupSQL"] = relationship(
```

Comment:
I wonder if this needs to be `Mapped[List["SiteGroupSQL"]]` or Mapped[["SiteGroupSQL"]], or something like that